docs/source/external_devices/payload_handling_area.rst

```diff
@@ -0,0 +1,284 @@
+Payload Handling Area
+=====================
+
+Payload Handling Areas are zones where actors are spawned, deleted, teleported, or altered. These areas are mainly used for:
+
+- Mimicking non-interested payload movements by spawning, deleting, and teleporting payloads in simulation.
+- Mimicking industrial machines by changing actors to another type in the simulation.
+
+Payload Handler Base Class
+--------------------------
+
+The `BP_PayloadHandlerBase` class serves as the base class for all payload handling area classes. It is a child class of `BP_ExternalDeviceBase`.
+
+Basic Functions
+^^^^^^^^^^^^^^^^
+
+- `/trigger` action is overwritten by child classes.  
+
+- **Manual and Auto Mode**:  
+
+  - *Manual*: The `/trigger` action is invoked manually by Unreal Engine (UE) or ROS 2.  
+  - *Auto*: The `/trigger` action is invoked periodically after a certain duration once a specific condition is met.  
+
+- **Target Actor Filtering**: The target actor is filtered by `TargetTag`.
+
+Payload Handler Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Param Name
+
+        [UE name if it is not pascal case of ROS one]
+     - Type (Default)
+     - Note
+   * -  **ROS JSON SPAWN PARAMETER**
+     - 
+     - 
+   * - /mode
+     - int32 (0)
+     - 0: Manual  
+       1: Auto
+   * - /duration
+     - float (1.0)
+     - Time interval to trigger action in auto mode.
+   * - /tag 
+
+        [TargetTag]
+     - string ('Payload')
+     - Filters target actors by this tag.
+
+ROS 2 API for Payload Handler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Topic Name
+     - Msg Type
+     - Note
+   * -  **SUBSCRIBE**
+     - 
+     - 
+   * - /set_mode
+     - `example_interfaces/msg/Int32 <https://docs.ros2.org/foxy/api/example_interfaces/msg/Int32.html>`_
+     - Sets the mode: 0 for manual, 1 for auto.
+   * - /set_duration
+     - `example_interfaces/msg/Float32 <https://docs.ros2.org/foxy/api/example_interfaces/msg/Float32.html>`_
+     - Sets the duration for auto-triggering actions.
+   * - /trigger
+     - `example_interfaces/msg/Empty <https://docs.ros2.org/foxy/api/example_interfaces/msg/Empty.html>`_
+     - Manually triggers an action.
+
+Sink Area
```

Comment:
nit: for me, "sink" has the image of element going down but it's not obvious the element gets deleted